src/lib.rs

```diff
@@ -800,9 +800,9 @@ pub trait BytesBody {
 }
 
 mod crypto {
-    #[cfg(all(feature = "aws-lc-rs", not(feature = "ring")))]
+    #[cfg(all(feature = "aws-lc-rs", any(feature = "fips", not(feature = "ring"))))]
     pub(crate) use aws_lc_rs as ring_like;
-    #[cfg(feature = "ring")]
+    #[cfg(all(feature = "ring", not(feature = "fips")))]
```

Comment:
I think this is worth some explanations in the README.
As I understand it:
If `ring` and `aws-lc-rs` are enabled: `ring` takes precedence unless `fips` is also enabled, in which case `aws-lc-rs` takes precedence.

Reply:
That's fair, added a commit for this.